tests/orm/nodes/data/test_data.py

```diff
@@ -154,7 +154,11 @@ def _generate_class_instance(data_class):
 
 @pytest.fixture(
     scope='function',
-    params=[entry_point for entry_point in plugins.get_entry_points('aiida.data') if entry_point.name != 'core.code'],
+    params=[
+        entry_point
+        for entry_point in plugins.get_entry_points('aiida.data')
+        if entry_point.name not in ('core.code', 'core.code.abstract')
```

Comment:
It's somewhat unfortunate that we're adding an entry point for an abstract class that cannot be instantiated. :-(

Reply:
I think it as okay. We also mention it in the RTD so it is an implementation detail we tried to hide from plugin developers so far.